1.2.0/index.html

```diff
@@ -292,191 +280,263 @@ <h2>Terminology</h2>
- in `archive/`. These files MUST use the `.cdxj` file extension and use the
- compressed <a>CDXJ</a> format. 
+The `indexes` directory MUST include one or more indexes for the raw data stored
+in `archive`. These files MUST use the `.cdx` file extension and use the
+compressed <a>CDXJ</a> format. 
+
+<pre class="example">
+indexes
+└── index.cdx
+</pre>
 
 ### pages.jsonl
 
-The `pages/pages.jsonl` file is a list of 'Page' objects as <a>JSONL</a> where  
-each line MUST contain at least the following fields:
+The `pages/pages.jsonl` MUST be present and include a list of 'Page' objects as
+[[JSON-Lines]] where each line MUST contain at least the following properties:
 
-- `url` - a valid URL (or URI/URN?)
-- `ts` - a valid ISO 8601 Date string
+- `url` - a URL for the page
+- `ts` - a [[RFC3339]] datetime string
 
-Each entry in the JSONL file MAY contain the following fields:
+Each entry in the [[JSONL]] file MAY contain the following properties to aid in
+navigating a web archive collection:
 
 - `title` - a string describing the resource
 - `id` - an arbitrary identifier for the resource
 - `text` - text extracted from the snapshot
 - `size` - an integer that representes the number of bytes for the page and all its resources
 
-Ex:
-```jsonl
+<pre class="example">
 {"format": "json-pages-1.0", "id": "pages", "title": "All Pages"}
 {"id": "1db0ef709a", "url": "https://www.example.com/page", "ts": "2020-10-07T21:22:36Z", "title": "Example Domain"}
 {"id": "12304e6ba9", "url": "https://www.example.com/another", "ts": "2020-10-07T21:23:36Z", "title": "Another Page"}
-```
-
-Other <a>JSONL</a> files MAY be added on using the same format in the `pages/` directory.
+</pre>
 
-For example, <a>py-wacz</a> supports specifying an 'extra' pages list, loaded from `extraPages.jsonl`.
+Each entry in the [[JSONL]] file MAY contain additional properties as long as
+they do not interfere with the required properties.
 
-A common use case is to include only the main pages in the `pages.jsonl`, while including additional pages, such as those discovered automatically
-via a crawl in an `extraPages.jsonl`.
+Other [[JSONL]] files MAY be added on using the same format in the `pages/`
+directory. A common use case is to include only the main pages in the
+`pages.jsonl`, while including additional pages, such as those discovered
+automatically via a crawl in an another file e.g. `extraPages.jsonl`.
 
 ### datapackage.json
 
-The `datapackage.json` file MUST be present which serves as the manifest for the
-web archive and is compliant with the [[FRICTIONLESS-DATA-PACKAGE]]
-specification. It MUST contain the following keys:
+The `datapackage.json` file MUST be present at the root of the WACZ which
+serves as the manifest for the web archive and is compliant with the
+[[FRICTIONLESS-DATA-PACKAGE]] specification. It MUST contain the following
+keys:
 
 - `wacz_version`: The version of WACZ being used. (e.g. 1.2.0)
```

Comment:
Should we remove this mention of `wacz_version` too, right?

Reply:
Yes, I meant to do that! Thanks for catching it.